src/wallet/rpcwallet.cpp

```diff
@@ -1396,7 +1396,7 @@ UniValue listtransactions(const JSONRPCRequest& request)
             "                                         'send' category of transactions.\n"
             "    \"confirmations\": n,       (numeric) The number of confirmations for the transaction. Negative confirmations indicate the\n"
             "                                         transaction conflicts with the block chain\n"
-            "    \"trusted\": xxx,           (bool) Whether we consider the outputs of this unconfirmed transaction safe to spend.\n"
+            "    \"trusted\": xxx,           (bool, optional) Whether we consider the outputs of this unconfirmed transaction safe to spend.\n"
```

Comment:
if this is optional, I think the help text should also mention when the value is present or not.